UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartTraumaDamage.cs

```diff
@@ -342,6 +368,37 @@ private void TakeBurnDamage(float burnDamage)
 			}
 		}
 
+
+		/// <summary>
+		/// Blunt damage is meant to reduce the bone/mechanical organs's health.
+		/// A less healthy bone is more likely to break and cause issues.
+		/// This damage should ONLY be applied to bodyParts that have CanBeBroken ticked.
+		/// </summary>
+		private void TakeBluntDamage(float bluntDamage)
+		{
+			if(CanBeBroken){health -= MultiplyTraumaDamage(bluntDamage);} //Incase we're damaging a mechanical body part.
```

Comment:
seems to be a strange place to change health, this isnt edited anywhere else in this file, could cause issues

Reply:
We're changing the health of organs and bodyParts that can be broken only, though thinking about it now; brute damage already does damage to health so the the first line in this function doesn't seem necessary and we should just stick to damaging organs inside body parts only.

Adjusting this tonight.